auth/auth.example.yml

```diff
@@ -3,3 +3,9 @@
   # WILL CAUSE ERRORS WHEN TRYING TO BUY/SELL
   gateio_api: "GATEIO_APIv4_KEY"
   gateio_secret: "GATEIO_SECRET_v4"
+
+  # Getting a bot token: https://core.telegram.org/bots#6-botfather
+  # Getting your chatID: find your bot and send a message,
+  # then go visit https://api.telegram.org/bot<YOUR_TOKEN>/getUpdates and find the chat ID
```

Comment:
When calling this with my token I get an empty result:
```
{"ok":true,"result":[]}
```
Is there some documentation about this?

Reply:
Did you actually send a message to your bot? This URL allows you to read incoming messages that the bot has not yet receied otherwise.

Reply:
See https://github.com/CyberPunkMetalHead/gateio-crypto-trading-bot-binance-announcements-new-coins/pull/18#discussion_r761246248.